src/main/scala/FbEvents.scala

```diff
@@ -6,17 +6,22 @@ import scalaj.http._
 
 object FbEvents {
   case class FbEvent(id: String, name: String, description: Option[String], startTime: ZonedDateTime, endTime: Option[ZonedDateTime]);
-  case class Response(data: List[FbEvent]);
+  case class Paging(next: Option[String]);
+  case class Response(data: List[FbEvent], paging: Option[Paging]);
 }
 
 trait FbEvents extends FbJsonMarshalling {
   import FbEvents._
 
-  def getEvents(token: String, pageId: String): List[FbEvent] = {
-    val url = s"https://graph.facebook.com/v2.7/$pageId/events?access_token=$token";
+  def getEvents(url: String): List[FbEvent] = {
     Http(url).asString match {
       case HttpResponse(body, 200, _) =>
-        body.parseJson.convertTo[Response].data
+        val response : Response = body.parseJson.convertTo[Response];
+        val currentResults : List[FbEvent] = response.data;
+        response.paging flatMap { _.next } match {
+           case None => currentResults
+           case Some(newUrl) => currentResults ::: getEvents(newUrl)
+        }
```

Comment:
I think this could be further simplified to something like:
```
        val response = body.parseJson.convertTo[Response]
        response.data ++ response.paging.flatMap(_.next) match {
                case None => Nil
                case Some(nextUrl) => getEvents(nextUrl)
        }                
```